### DISCUSSION TITLE
Will it work on Tx2i

### DISCUSSION THREAD

**Discussion by vijaypurohit322** [pythops/jetson-image#107] [Q&A]
Hi, I want minimalist, Ubuntu-based images for the Nvidia jetson Tx2i board, will it work for the same?

**Comment by pythops** [comment]
It's not supported for the moment, but you're welcome to make a PR to add support for it.

**Reply by vijaypurohit322** [reply]
Sure, I'll do the needful!

Thanks @pythops!
